Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -764,6 +766,17 @@ type AddVehicleForFleetAPI =
 
 ---------------------------------------------------------
 
+-- add vehicle for fleet withoutDriver ------------------------------------------
+
+type RegisterRCForFleetWithoutDriverAPI =
+  "fleet"
+    :> "addRC"
+    :> "withoutDriver"
+    :> ReqBody '[JSON] AddVehicleReq
```

Comment:
Why request type is `AddVehicleReq`? Why do we want from user to pass color, make, model etc?

Reply:
fixed